brainstormforce/ultimate-addons-for-gutenberg
# Fix: Block Icons - Resolved icons look weird in list view.

### Description

- https://app.startinfinity.com/b/Mi5zu3VRVzk/M7Dpq9yR4Pe/9a049355-9ce0-4e15-8e56-cf40b70afbfc?redirect=true&view=6d3bcc3d-b047-4739-8e05-d9621e9f9445&t=attributes

### Screenshots

- [Screencast 1](https://share.getcloudapp.com/P8u191nY)
- [Screencast 2](https://share.getcloudapp.com/7KuQo6gA)

### Types of changes




- Bug fix (non-breaking change which fixes an issue)

### How has this been tested?

- Drag and drop blocks on the page.
- Check List view refer to Screencast 1.
- Check Select view refer to Screencast 2.

### Checklist:
